pixart.js

```diff
@@ -1 +1,21 @@
+var brushBox = document.querySelector(".brush");
+var inputBox = document.innerHTML.querySelector("#color-field").value;
+var colorButton = document.querySelector("#set-color")/
 
+function setColor() {
+  document.body.style.background = "#1B4370";
```

Comment:
This line is a good start, but note that it's changing the background color of the body's background, not of the 'brush'.